src/picongpu/include/simulation_defines/unitless/ionizerConfig.unitless

```diff
@@ -22,3 +22,10 @@
 
 #include "particles/ionization/byField/BSI/AlgorithmBSI.hpp"
 #include "particles/ionization/None/AlgorithmNone.hpp"
+
+namespace picongpu
+{
+    /* magic number for ionization probability calculation to reduce computation time */
+    const float_X PROB_THRESH = 0.1;
```

Comment:
you should not write that this *is* a magic number but *explain what this constant controls*.

I do not understand the physical meaning of it, pls elaborate a bit more :)